201-blob-storage-private-endpoint/README.md

```diff
@@ -54,10 +88,46 @@ sudo apt-get update -y
 # Upgrade packages
 sudo apt-get upgrade -y
 
+# Install curl and traceroute
+sudo apt install -y curl traceroute
+
+# Install Azure CLI
+curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+
 # Run nslookup to verify that the <storage-account>.blob.core.windows.net public hostname of the storage account 
 # is properly mapped to <storage-account>.privatelink.blob.core.windows.net by the private DNS zone
 # and the latter mapped to the private address by the A record
-nslookup $blobServicePrimaryEndpoint
+nslookup "$storageAccountName.blob.core.windows.net"

```

Comment:
I think we should remove the code from the readme...  it will just make it a maintenance issue - in this case you're not actually hard-coding the suffix anymore (in the script) but it still shows here.

Reply:
You are right, sorry, done :)

Reply:
I'm still seeing the script here - did you not push yet?

Reply:
I fixed the script yesterday but forgot to update the README file. Now it's fixed, thanks.